core/templates/external-js/save-all-external-js.tid

```diff
@@ -5,7 +5,10 @@ title: $:/core/save/all-external-js
 \define saveTiddlerFilter()
 [is[tiddler]] -[prefix[$:/state/popup/]] -[prefix[$:/temp/]] -[prefix[$:/HistoryList]] -[status[pending]plugin-type[import]] -[[$:/core]] -[[$:/boot/boot.css]] -[type[application/javascript]library[yes]] -[[$:/boot/boot.js]] -[[$:/boot/bootprefix.js]] +[sort[title]] $(publishFilter)$
 \end
+<!-- It's important to fully understand this PR https://github.com/Jermolene/TiddlyWiki5/pull/5570 -->
+<!-- and what tiddlers: $:/core/templates/tiddlywiki5.js and $:/core/templates/tiddlywiki5.js/tiddlers do -->
```

Comment:
These comments are very cryptic. What are we trying to accomplish here? If there's a relevant piece of information I'd rather we provided it directly

Reply:
Line 10 has been introduced with the PR #5570 and it's not obvious, what it does. The variable is named `defaultCoreURL` and the tiddler is named `core/templates/tiddlywiki5.js` ... Templates are usually used to create HTML output. .. So it's important to understand what's going on here. ... 

Additionally there are some inconsistencies between filenames and the tiddler titles ... So all of this has potential for confusion. ... Finding the connection between the different inconsistencies did cost me several hours of reverse engineering. The link to the PR and the 2 tiddler names are the best starting point.

Reply:
I could shorten the comment to 1 line eg:
`<!-- see and understand $:/core/templates/tiddlywiki5.js, $:/core/templates/tiddlywiki5.js/tiddlers and PR #5570 ! -->`

But IMO that's even more cryptic.